src/rust/ide/view/graph-editor/src/lib.rs

```diff
@@ -2459,6 +2512,22 @@ fn new_graph_editor(app:&Application) -> GraphEditor {
    }));
 
 
+   // === Vis Selection ===
+    frp::extend! { network
+        eval out.on_visualization_select ([model](switch) {
+            if switch.is_on() {
+                model.visualisations.selected.insert(switch.value);
+            } else {
+                model.visualisations.selected.remove(&switch.value);
+            }
+        });
+
+        selection_change <- out.on_visualization_select.constant(());
```

Comment:
this .constant is not needed. You are discarding the value in next line again, so this is just unnecesary FRP node = performance waste